VideoLocker/src/main/java/org/edx/mobile/social/SocialLoginDelegate.java

```diff
@@ -211,21 +213,25 @@ public ProfileModel call( ) {
 
                 // do SOCIAL LOGIN first
                 AuthResponse social = null;
+                HashMap<String, CharSequence> descParams = new HashMap<>();
+                String platformName = environment.getConfig().getPlatformName();
+                descParams.put("platform_name", environment.getConfig().getPlatformName());
+                descParams.put("platform_destination", environment.getConfig().getPlatformDestinationName());
```

Comment:
Should this mapping be stored in some global area (maybe a utility method in `Config` or `ResourceUtil`), so that it can be reused?